app/client/templates/elements/compileContract.html

```diff
@@ -53,11 +53,11 @@
                             <select class="compiled-contracts">
                                 <option disabled selected>{{i18n "wallet.contracts.pickContract"}}</option>
                                 {{#each this}}
-                                    <option value="{{name}}">{{toSentence name true}}</option>
+                                    <option value="{{name}}" selected="{{selected}}">{{toSentence name true}}</option>
```

Comment:
It turns out blaze behaves differently from _regular_ dom api on this matter. That won't be a problem. See [blaze selected attribute issue](https://github.com/meteor/meteor/issues/3191)